packages/cli/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cli",
+  "version": "1.0.0",
+  "description": "A React-Navigation CLI",
+  "main": "build/cli.js",
+  "bin": {
+    "rnav": "build/cli.js"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepare": "npm run build",
+    "build": "npx tsc"
+  },
+  "dependencies": {
+    "commander": "^6.2.0",
+    "enquirer": "^2.3.6",
+    "node-fetch": "^2.6.1",
+    "ora": "^5.4.1",
+    "shelljs": "^0.8.4"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.2",
+    "@types/node": "^16.9.4",
+    "@types/shelljs": "^0.8.9",
+    "jest": "^27.2.1",
+    "memfs": "^3.3.0",
+    "ts-jest": "^27.0.5",
+    "ts-node": "^9.0.0",
+    "typescript": "^4.4.3"
```

Comment:
We shouldn't duplicate deps such as `typescript` or `jest` here as they are in the root project already. Also `ts-jest` doesn't seem necessary here.